### DISCUSSION TITLE
more power saving mode

### DISCUSSION THREAD

**Discussion by venus13v** [seerge/g-helper#1071] [Ideas]
i have a tuf dash f15 laptop and i notced that the balanced mod is the best for power saving and it alomst have no diffrence to the silent mod <<<
 1)I suggest Editing the Silent mod for more power saving stop unnessery survices that consume more power or edit it to stop background apps 
2) is there a way to make the app power saving the intel internal graphics too  like undervolting 
3) you caan make a new mod thats called (ultra saving ) and start experment on it . ultra will power save in everything in windows >apps , browsers , internal graphics , video players , ect
4) I notced in task manager that comes with windows 11 there is a feature called efficiency mod but u can activate that manually but it's so huge it can actually saves alot i activate that every time on browers video players and every time manually Im sure you can add that or penevent from it

**Comment by IceStormNG** [comment]
I think you misunderstand the performance modes. Silent, balanced and turbo is not about power saving, but mainly about fan noise.
While they do change the performance levels, it is a side effect of limited cooling.

These modes will do exactly one thing: They change the performance mode of the laptop in the firmware, and optionally change the windows powerplan/overlay.

Windows efficiency mode will simply throttle the app and will be done my windows automatically on battery when an app draws too much power in background. Some Apps can also apply it themselves or can be set in the Windows App Settings.

I personally would say that this is out of scope. Most people use GHelper because it is lighter and only does what the button says and not 1000 different things in background that you have no control of or is not documented., which is what Armoury Crate does.

**Reply by venus13v** [reply]
since i used Ghelper my battery life doubled why it goes from 2h30min to 4 h why is that? since Ghelper ddoesnt give you a power saving
